pennylane/operation.py

```diff
@@ -800,6 +800,97 @@ def _matrix(cls, *params):
         return np.diag(cls._eigvals(*params))
 
 
+class Channel(Operation):
+    r"""Base class for quantum channels.
+
+    As with :class:`~.Operation`, the following class attributes must be
+    defined for all channels:
+
+    * :attr:`~.Operator.num_params`
+    * :attr:`~.Operator.num_wires`
+    * :attr:`~.Operator.par_domain`
+
+    The following two class attributes are optional, but in most cases
+    should be clearly defined to avoid unexpected behavior during
+    differentiation.
+
+    * :attr:`~.Operation.grad_method`
+    * :attr:`~.Operation.grad_recipe`
+
+    To define a noisy channel, the following attribute of :class:`~.Channel`
+    can be used to list the corresponding Kraus matrices.
+
+    * :attr:`~.Channel._kraus_matrices`
+
+    Args:
+        params (tuple[float, int, array, Variable]): operation parameters
+
+    Keyword Args:
+        wires (Sequence[int]): Subsystems the channel acts on. If not given, args[-1]
+            is interpreted as wires.
+        do_queue (bool): Indicates whether the operation should be
+            immediately pushed into a :class:`BaseQNode` circuit queue.
+            This flag is useful if there is some reason to run an Operation
+            outside of a BaseQNode context.
+    """
+    # pylint: disable=abstract-method
+
+    @classmethod
+    def _kraus_matrices(cls, *params):
+        """Kraus matrices representing a quantum channel, specified in
+        the computational basis.
+
+        This is a class method that should be defined for all
+        new channels. It returns the Kraus matrices representing
+        the channel in the computational basis.
+
+        This private method allows matrices to be computed
+        directly without instantiating the channel first.
+
+        **Example**
+
+        >>> qml.AmplitudeDamping._kraus_matrices(0.1)
+        >>> [array([[1.       , 0.       ],
+        [0.       , 0.9486833]]), array([[0.        , 0.31622777],
+        [0.        , 0.        ]])]
+
+        To return the Kraus matrices of an *instantiated* channel,
+        please use the :attr:`~.Operator.kraus_matrices` property instead.
+
+        Returns:
+            list(array): list of Kraus matrices
+        """
+        raise NotImplementedError
+
+    @property
+    def kraus_matrices(self):
+        r"""Kraus matrices of an instantiated channel
+        in the computational basis.
+
+        ** Example**
+
+        >>> U = qml.AmplitudeDamping(0.1, wires=1)
+        >>> U.kraus_matrices
+        >>> [array([[1.       , 0.       ],
+        [0.       , 0.9486833]]), array([[0.        , 0.31622777],
+        [0.        , 0.        ]])]
+
+        Returns:
+            list(array): list of Kraus matrices
+        """
+        return self._kraus_matrices(*self.parameters)
+
+    def __init__(self, *params, wires=None, do_queue=True):
+
+        self._inverse = False
```

Comment:
Seems like this def will occur when [calling `super().__init__` ](https://github.com/PennyLaneAI/pennylane/blob/781b322aba9a3f34353a2fc3a2db631b41870a15/pennylane/operation.py#L690) (let me know if missing something!)
```suggestion
```